src/detection/disk/disk_linux.c

```diff
@@ -73,9 +73,9 @@ void ffDetectDisksImpl(FFDiskResult* disks)
         #endif
 
         //Detects stats
-        struct statvfs fs;
-        if(statvfs(disk->mountpoint.chars, &fs) != 0)
-            memset(&fs, 0, sizeof(struct statvfs)); //Set all values to 0, so our values get initialized to 0 too
+        struct statvfs64 fs;
+        if(statvfs64(disk->mountpoint.chars, &fs) != 0)
+            memset(&fs, 0, sizeof(struct statvfs64)); //Set all values to 0, so our values get initialized to 0 too
```

Comment:
I would put a
```c
#ifndef _LARGE_FILE_API
    #define _LARGE_FILE_API
#endif
```
at the top of the file to ensure the prototype is available on all platforms.

Reply:
Well, should it be defined by glibc? If it's not defined, it might be disabled when configuring.

Reply:
Ah right. We might switch between `statvfs64` and `statvfs` at compile time, depending weather its defined or not.

Reply:
Ok